ceph_cfg/tests/test_update_lsblk_version.py

```diff
@@ -0,0 +1,93 @@
+#Test units
```

Comment:
nit, maybe if there is a # on the first line it should be a shebang #!/usr/env python maybe?